doc/source/skips/1-values.rst

```diff
@@ -0,0 +1,168 @@
+======================================
+SKIP-1: scikit-image mission statement
+======================================
+
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Status: Active
+:Type: Process
+:Created: 2018-12-08
+:Resolved:
+:Resolution:
+:Version effective: 0.16
+
+Abstract
```

Comment:
Something we've been wanting to change for NumPy too: the SKIP template should probably ask for an abstract, and then a motivation section.

I don't think the abstract or the motivation should be opinionated: "should adopt this document"